sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/BatchMessageOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    /// <summary>
+    /// Configuration options for functions that bind to multiple multiple <see cref="ServiceBusReceivedMessage"/>.
+    /// </summary>
+    public class BatchMessageOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum number of messages that will be passed to each function call. The default value is 1000.
+        /// </summary>
+        public int MaxMessages { get; set; } = 1000;
```

Comment:
Seems odd that there's an exception handler for the single but not the batch...  was that intentional?

Reply:
Yeah this is just a side effect of the single using the Processor whereas batch uses the Receiver.